app/services/floating_ip_address_visibility_service.rb

```diff
@@ -0,0 +1,14 @@
+class FloatingIpAddressVisibilityService
+  def determine_visibility(floating_ip_address, number_of_vms)
```

Comment:
`floating_ip_address` isn't being used at all here, is it supposed to be? If not, we can probably just completely scrap this whole visibility service and add the logic for hiding/editing the `:floating_ip_address` field into the `NumberOfVmsVisibilityService`.